.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+def generate_bar_chart(value, width):
```

Ruff (0.11.9)

Missing return type annotation for public function `generate_bar_chart`

Add return type annotation: `None`

(ANN201)